.github/workflows/lock-closed-issues-workflow.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Lock threads'
```

Comment:
I suggest to rename it to "Lock inactive threads". This only closes inactive threads which were already closed, right?

Reply:
Fixed.